app/shared/models/domain/Status.scala

```diff
@@ -16,19 +16,23 @@
 
 package shared.models.domain
 
-import play.api.libs.json.Format
-import shared.utils.enums.Enums
+import play.api.libs.json.*
+import shared.definition.APIStatus
 
-sealed trait Status {}
+enum Status {
+  case valid, invalid, superseded
+}
 
-//noinspection ScalaStyle
 object Status {
-
-  case object valid extends Status
-
-  case object invalid extends Status
-
-  case object superseded extends Status
-
-  implicit val format: Format[Status] = Enums.format[Status]
+  given Format[Status] = Format(
+    Reads {
+      case JsString(status) => Status.values.find(_.toString == status).map(JsSuccess(_))
+        .getOrElse(JsError(s"Unknown Status: $status"))
+      case _ => JsError("Expected a string for Status")
+    },
+    Writes {
+      (status: Status) => JsString(status.toString)
+      }
+    )
+  
```

Comment:
Same comment around the enums. See [here](https://github.com/hmrc/individuals-reliefs-api/commit/8d5ec9fe06ffdad860876b9e2e7d50a544d5c0c4#diff-a4f4f8a81fed23cb322a17b96f2b267702b67f3b615d74b0cdd36ffd7877ccbd) for update, thanks.